eventorder_test.go

```diff
@@ -98,48 +103,46 @@ func TestEventOrderGuarantee(t *testing.T) {
 		"workspaceId": workspaceID,
 	}
 	configJsonPath := workspaceConfig.CreateTempFile(t, "testdata/eventOrderTestTemplate.json", templateCtx)
-	t.Setenv("RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE", "true")
-	t.Setenv("RSERVER_BACKEND_CONFIG_CONFIG_JSONPATH", configJsonPath)
+	config.Set("BackendConfig.configFromFile", true)
```

Comment:
what is the  reason we are not using `t.Setenv `anymore ?

Reply:
We cannot consider having tests running in parallel at any point in the future if we continue using environment variables for setting up our test environments. `t.Setenv` sets an environment variable and unsets it when the test is done, but this doesn't mean that the environment variable is specific for the test, it will affect all other tests running in parrallel at the same time.